### PR TITLE
[ar] Translate setup/_index.md

### PR DESCRIPTION
**Pull request by essamgouda97**
This PR will create a new file contents/docs/ar/setup/_index.md that is translated into arabic following the english docs.
Following the plan in this PR: https://github.com/kubernetes/website/issues/37546 more file will be picked. The base branch is currently `main` until a branch is created for arabic.

### CODE REVIEW

**Comment by adowair** [comment]
Thank you @essamgouda97 for this effort! I cannot seem to find a link that will show me a preview of this page. Have you been able to find it? https://deploy-preview-44289--kubernetes-io-main-staging.netlify.app/ takes us to the homepage, but I'm not able find this page manually by manipulating the URL.

File: content/ar/docs/setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- نظره عامه -->
+
+ هذا القسم يعرض الطرق المختلفه لاعداد وتشفيل كوبرنيتس. عندما تقوم بتثبيت كوبرنيتس، اختر نوع التثبيت بناء على: سهولة الصيانة، الأمان، التحكم، الموارد المتاحة، والخبرة المطلوبة لتشغيل وإدارة عنقود.
```
Comment (adowair) [inline content/ar/docs/setup/_index.md:19]:
```suggestion
 هذا القسم يعرض الطرق المختلفه لإعداد وتشفيل كوبرنيتس. عندما تقوم بتثبيت كوبرنيتس، اختر نوع التثبيت بناء على: سهولة الصيانة، الأمان، التحكم، الموارد المتاحة، والخبرة المطلوبة لتشغيل وإدارة عنقود.
```

File: content/ar/docs/setup/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: بدء الاستخدام
```
Comment (adowair) [inline content/ar/docs/setup/_index.md:2]:
```suggestion
title: دليل الاستخدام
```

File: content/ar/docs/setup/_index.md (hunk 1 above)
Comment (adowair) [inline content/ar/docs/setup/_index.md:19]:
```suggestion
 يعرض هذا القسم الطرق المختلفه لاعداد وتشفيل كوبرنيتس. إختر الطريقة المناسبة للتثبيت بناءً على متطلبات الصيانة والأمان والتحكم، والموارد المتاحة للتشغيل، والخبرة التقنية المتوفرة لإدارة العنقود.
```

File: content/ar/docs/setup/_index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ هذا القسم يعرض الطرق المختلفه لاعداد وتشفيل كوبرنيتس. عندما تقوم بتثبيت كوبرنيتس، اختر نوع التثبيت بناء على: سهولة الصيانة، الأمان، التحكم، الموارد المتاحة، والخبرة المطلوبة لتشغيل وإدارة عنقود.
+
+يمكنك [تنزيل كوبرنيتس](/releases/download/) لنشر عنقود كوبرنيتس على جهاز محلي، أو في السحابة، أو لمركز البيانات الخاص بك.
+
+يمكن أيضًا تشغيل عدة مكونات [كوبرنيتس](/docs/concepts/overview/components/) مثل {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}} أو {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} [كصور الحاويات](/releases/download/#container-images) داخل العنقود.
+
+**من المستحسن** تشغيل مكونات كوبرنيتس كصور حاويات أينما كان ذلك ممكنًا، والسماح لكوبرنيتس بإدارة تلك المكونات. لا يمكن تضمين المكونات التي تعمل على تشغيل الحاويات - على وجه الخصوص، kubelet - في هذه الفئة.
+
+
+إذا كنت لا تريد إدارة عنقود كوبرنيتس بنفسك، يمكنك اختيار خدمة مُدارة، بما في ذلك [المنصات المعتمدة](/docs/setup/production-environment/turnkey-solutions/). هناك أيضًا حلول موحدة ومخصصة أخرى عبر مجموعة واسعة من البيئات السحابية والمعدنية الخام.
```
Comment (adowair) [inline content/ar/docs/setup/_index.md:28]:
```suggestion
إذا كنت لا تريد إدارة العنقود بنفسك، يمكنك اختيار خدمة مُدارة، بما في ذلك [المنصات المعتمدة](/docs/setup/production-environment/turnkey-solutions/). هناك أيضًا حلول موحدة ومخصصة أخرى عبر مجموعة واسعة من البيئات السحابية والمعدنية الخام.
```

**Review by adowair — CHANGES_REQUESTED** [review]
شكراً على المشاركة @essamgouda97، أقترح إعادة صياغة الترجمة حيث أنها قريبة جداً إلى الترجمة الآلية. اقترح قراءة النص الإنجليزي كاملاً إستخراج المعاني والأهداف ومن ثم تحرير الترجمة بأسلوبك الخاص.

**Comment by essamgouda97** [comment]
> Thank you @essamgouda97 for this effort! I cannot seem to find a link that will show me a preview of this page. Have you been able to find it? https://deploy-preview-44289--kubernetes-io-main-staging.netlify.app/ takes us to the homepage, but I'm not able find the page of this PR manually by manipulating the URL.

@adowair any idea what's missing ? How does that work anyways 😆 ?

File: content/ar/docs/setup/_index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- نظره عامه -->
+
+ يعرض هذا القسم الطرق المختلفه لاعداد وتشفيل كوبرنيتس. إختر الطريقة المناسبة للتثبيت بناءً على متطلبات الصيانة والأمان والتحكم، والموارد المتاحة للتشغيل، والخبرة التقنية المتوفرة لإدارة العنقود.
```
Comment (mboukhalfa) [inline content/ar/docs/setup/_index.md:19]:
```suggestion
 يعرض هذا القسم الطرق المختلفه لاعداد وتشفيل كوبرناتيز. إختر الطريقة المناسبة للتثبيت بناءً على متطلبات الصيانة والأمان والتحكم، والموارد المتاحة للتشغيل، والخبرة التقنية المتوفرة لإدارة العنقود.
```

File: content/ar/docs/setup/_index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+يمكنك [تنزيل كوبرنيتس](/releases/download/) لنشر عنقود كوبرنيتس على جهاز محلي، أو في السحابة، أو لمركز البيانات الخاص بك.
```
Comment (mboukhalfa) [inline content/ar/docs/setup/_index.md:21]:
```suggestion
يمكنك [تنزيل كوبرناتيز](/releases/download/) لنشر عنقود كوبرناتيز على جهاز محلي، أو في السحابة، أو لمركز البيانات الخاص بك.
```

File: content/ar/docs/setup/_index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+يمكن أيضًا تشغيل عدة مكونات [كوبرنيتس](/docs/concepts/overview/components/) مثل {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}} أو {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} [كصور الحاويات](/releases/download/#container-images) داخل العنقود.
```
Comment (mboukhalfa) [inline content/ar/docs/setup/_index.md:23]:
```suggestion
يمكن أيضًا تشغيل عدة مكونات [كوبرناتيز](/docs/concepts/overview/components/) مثل {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}} أو {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} [كصور الحاويات](/releases/download/#container-images) داخل العنقود.
```

File: content/ar/docs/setup/_index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+**من المستحسن** تشغيل مكونات كوبرنيتس كصور حاويات أينما كان ذلك ممكنًا، والسماح لكوبرنيتس بإدارة تلك المكونات. لا يمكن تضمين المكونات التي تعمل على تشغيل الحاويات - على وجه الخصوص، kubelet - في هذه الفئة.
+
+
+إذا كنت لا تريد إدارة العنقود بنفسك، يمكنك اختيار خدمة مُدارة، بما في ذلك [المنصات المعتمدة](/docs/setup/production-environment/turnkey-solutions/). هناك أيضًا حلول موحدة ومخصصة أخرى عبر مجموعة واسعة من البيئات السحابية والمعدنية الخام.
+
+<!-- الجسم -->
```
Comment (mboukhalfa) [inline content/ar/docs/setup/_index.md:30]: do not think we translate this

**Review by mboukhalfa — COMMENTED** [review]
Great work here are some comments

**Comment by mboukhalfa** [comment]
you can squash commits and move from draft to ready for review

File: content/ar/docs/setup/_index.md (hunk 7 above)
Comment (essamgouda97) [reply content/ar/docs/setup/_index.md:30]: what should be the solution here ? Do we ignore or refer to the english docs until we translate ?

Comment (essamgouda97) [reply content/ar/docs/setup/_index.md]: my bad coffee is still kicking in, I mistakened your comment line 😆

**Comment by essamgouda97** [comment]
@mboukhalfa @adowair I want one more shot at improving the language used this weekend then I will squad and mark as Ready for review.
After that I will move on to another page.

File: content/ar/docs/setup/_index.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- General overview-->
```
Comment (mboukhalfa) [inline content/ar/docs/setup/_index.md:17]: in the english version it was just overview
https://github.com/kubernetes/website/blob/main/content/en/docs/setup/_index.md?plain=1
TBH not sure if this comments are interpreted by the template engine to format the page so let's keep it as it was in the english version

**Comment by essamgouda97** [comment]
@mboukhalfa @adowair this being my first OSS contribution ever I am bit noob-ish, I rebased off `dev-1.29-ar.1` but there are files that I didn't edit that are there, is the branch not up to date with main ? My PR is ready for review I will mark it as such once I got it right.

**Comment by mboukhalfa** [comment]
> @mboukhalfa @adowair this being my first OSS contribution ever I am bit noob-ish, I rebased off `dev-1.29-ar.1` but there are files that I didn't edit that are there, is the branch not up to date with main ? My PR is ready for review I will mark it as such once I got it right.

I cannot see what is really causing this probably you are not forking the correct branch ?  fork `dev-1.29-ar.1` create a branch from from it then add you changes better in a single commit if not necessary separate you commits then create a PR from that branch to dev-1.29-ar.1 upstream

**Comment by adowair** [comment]
This PR is superseded by #45094

**Pull request closed without merging**